modules/simulator.js

```diff
@@ -72,8 +77,36 @@ function createOrganism(land)
     var randomMass = Constant.MIN_MASS + (Math.random()*Math.random()) * Constant.MAX_MASS;
     var randomMinSpeed =  (Math.random()) + Constant.MIN_SPEED;
 
-    // create Organism(generation, mass, energy, x, y, vision_angle, vision_range, minSpeed, fertility)
-    return new Organism(1, randomMass, randomX, randomY, 100, 100, randomMinSpeed, 1);
+    // 
+    if(organismCounter%3 == 0) {
```

Comment:
Right switch might be better! and yeah modulo was put there initially because I wasn't using randomized number function but more of a counter so it goes all the way up to 0 to 9 (due to population count) will fix it right away!